### PR TITLE
[release-5.8] LOG-5449: Address CVE-2023-45288

### PR DESCRIPTION
**Pull request by periklis**
### Description
Address CVE-2023-45288 in `release-5.8`

/cc @jcantrill @xperimental

### CODE REVIEW

File: Dockerfile.in [hunk 1]
```diff
@@ -1,5 +1,6 @@
-#@follow_tag(registry-proxy.engineering.redhat.com/rh-osbs/openshift-golang-builder:rhel_9_golang_1.20)
-FROM registry-proxy.engineering.redhat.com/rh-osbs/openshift-golang-builder:v1.20.3-202306041558.el9.gee76e36 AS builder
+### This is a generated file from Dockerfile.in ###
```
Comment (vparfonov) [inline Dockerfile.in:1]: nit: redundant line, duplicated Dockerfile.dev

**Review by jcantrill — COMMENTED** [review]
/approve
/lgtm

**Pull request merged**